test/hex.js

```diff
@@ -30,15 +30,24 @@ describe("to.rpcQuantityHexString", function() {
   });
 });
 
-function noLeadingZeros(result) {
+function noLeadingZeros(method, result, path) {
```

Comment:
This test needs improvement, as it sometimes fails when fixed-width fields naturally have their most significant nibble zeroed out (e.g. hashes, etc). Further, if taken as documentation/specification, it's just plain wrong. It's not that we care about leading zeros, it's that we care about whether the field is fixed width or not.

further improvement is out of scope for this PR, however. I just added these changes because the test failed on me on one of the runs, and I wanted it to print out a better message next time.